.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+Thanks for contributing! Please remove any top-level sections that do not apply to your changes.
+
+- [ ] `make format` and `make documentation` has been run. (You may also want to run `make test`.)
+
+# New variable
+
+- [ ] Label field added
```

Comment:
I think some of these checklist items can be removed as they are not relevant to OG-ZAF

Reply:
@jdebacker. I removed this section from the `PULL_REQUEST_TEMPLATE.md` file.